orchagent/orchdaemon.cpp

```diff
@@ -621,9 +621,12 @@ bool OrchDaemon::init()
 
     m_orchList.push_back(&CounterCheckOrch::getInstance(m_configDb));
 
-    vector<string> p4rt_tables = {APP_P4RT_TABLE_NAME};
-    gP4Orch = new P4Orch(m_applDb, p4rt_tables, vrf_orch, gCoppOrch);
-    m_orchList.push_back(gP4Orch);
+    if (platform == P4_PLATFORM_SUBSTRING)
```

Comment:
The p4orch should be able to initialized successfully on platform=broadcom switches. And the default UDF match created in p4orch during init is calling standard SAI API(https://github.com/Azure/sonic-swss/blob/eff725ff1803f7e4948a25e53abc68886e43b367/orchagent/p4orch/acl_table_manager.cpp#L40)